docs/plans/2026-03-09-call-parameters-discarded.md

```
@@ -0,0 +1,256 @@
+
+Work from the repository root `/data/leynos/Projects/ironclaw`.
+
+1. Confirm the current placeholder path and override asymmetry:
+
+```plaintext
+nl -ba src/tools/wasm/runtime.rs | sed -n '260,355p'
+nl -ba src/tools/wasm/loader.rs | sed -n '145,170p'
+nl -ba src/tools/registry.rs | sed -n '643,675p'
+```
+
+2. Add the first failing test showing that a file-loaded real WASM tool publishes the placeholder schema externally.
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)